tests/test_prompt_chain.py

```diff
@@ -25,6 +25,29 @@ def describe_weather(city: str) -> str:
     assert isinstance(output, str)
 
 
+@pytest.mark.openai
+def test_prompt_chain_max_calls():
```

Comment:
Could you use a mock for the model here and remove the openai mark since this doesn't need to query openai. That way this will be run as part of the automated tests. For an example of using mocking see https://github.com/jackmpcollins/magentic/blob/d207148785e48b33e268e8d45fd227cfb5336e92/tests/test_chatprompt.py#L73-L89